tests/performance/set_index.xml

```diff
@@ -14,7 +14,14 @@
     </stop_conditions>
```

Comment:
you can also add an e2e test with `force_primary_key` like those in https://github.com/yandex/ClickHouse/blob/master/dbms/tests/queries/0_stateless/00562_in_subquery_merge_tree.sql